content/docs/command-reference/exp/branch.md

```diff
@@ -6,22 +6,24 @@ branch.
 ## Synopsis
 
 ```usage
-usage: dvc exp branch [-h] [-q | -v] experiment branch
+usage: dvc exp branch [-h] [-q | -v] experiment [branch]
 
 positional arguments:
   experiment     Experiment to turn into a branch
-  branch         Name for the new Git branch
+  branch         Optional name for the new Git branch. Defaults to 'exp-{experiment-name}'
 ```
 
 ## Description
 
-Creates a new [Git branch] containing the target `experiment`. The given
-`branch` name will be used. It will stem from the experiment's baseline (`HEAD`
-at the time the experiment was run).
+Creates a new [Git branch] containing the target `experiment` from the
+experiment's baseline (`HEAD` at the time the experiment was run).
```

Comment:
```suggestion
Creates a new [Git branch] containing the target `experiment` from the
experiment's baseline (`HEAD` at the time the experiment was run).
```

Unrelated/not blocking: I'm not sure what the second part of this sentence means and would be fine to drop it if you don't think it's helpful.